composer.json

```diff
@@ -29,17 +29,17 @@
         }
     ],
     "require": {
-        "php": ">=7.4.0",
-        "cakephp/cakephp": "^4.4.1",
-        "friendsofcake/crud": "^6.0",
-        "laravel-json-api/neomerx-json-api": "^5.0"
+        "php": ">=8.1.0",
+        "cakephp/cakephp": "^5.0.0",
+        "friendsofcake/crud": "^7.2.0",
```

Comment:
Is this ok? It seems needed due to this change:

https://github.com/FriendsOfCake/crud/pull/754/files#diff-5aa78d26d9d06925cb6d1434c1c8afa3962704f1041ef3ecf1cb25561e66ee43L124